README.md

```diff
@@ -1,45 +1,75 @@
-# Catapult-explorer
+# catapult-explorer
 
-## Setup Client
-```
-cd client
-npm install
-```
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-#### Compiles and hot-reloads for development
-```
-cd client
-npm run serve
-```
+The nem2 Command Line Interface (CLI) is a unified tool to interact with the NEM2 Blockchain.
 
-#### Compiles and minifies for production
-```
-cd client
-npm run build
-```
+This tool will enable you perform the most common and regular used actions in NEM2 (a.k.a Catapult)
+
+
+
+## Requirements
+
+The catapult-explorer works on NodeJS versions:
+
+- 8.9.X
+- 9.X.X
+
+## Installation
+
+Client
 
-#### Lints and fixes files
 ```
 cd client
-npm run lint
+
+npm install // install related package.
 ```
 
-## Setup API Server
+Server
+
 ```
 cd server
-npm install
-```
 
-#### Setup environment
-```
+npm install // install related package.
+
 cp .env.example .env
 
 ** Configure all required variables in .env
```

Comment:
A table describing each required variable would be helpful. If you consider that it is not necessary at the moment, please link to the file.